src/app/components/Header/index.jsx

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
-import Brand from '../Brand';
+import BrandContainer from '../../containers/BrandContainer';
```

Comment:
I think components importing containers is an antipattern/bad smell, given we're about to try to pull out as many components as possible.

Reply:
@bcmn @sareh Ok I agree. And by this line of reasoning the Footer component also needs to be refactored, so that the Footer component is pulling in `<BrandContainer />`?